source/install/upgrade_from_mongodb.rst

```diff
@@ -17,38 +18,46 @@ An in-place upgrade is done with existing data in the server.  Generally
 speaking, this is stopping the server, removing the old packages, installing the
 new server and starting it with the same data files.  While an in-place upgrade
 may not be suitable for high-complexity environments, it should work in most
-cases.
+cases.  
 
 .. warning::
 
-   Before starting the upgrade process it is recommended to perform a full
+   Before starting the upgrade service, we recommend to perform a full
```

Comment:
I guess the 'upgrade process' was used with meaning of 'upgrade procedure' here. So it seems 'service' should not be used here

Reply:
Yes, "upgrade service" isn't used. "Upgrade procedure" is the most normal phrase I can think of. "Upgrade process" can be used too but procedure sounds better.